app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -193,6 +193,9 @@ $autocompleteApiKey = $algoliaHelper->generateSearchSecuredApiKey($config->getSe
 $algoliaJsConfig = array(
     'instant' => array(
         'enabled' => $config->isInstantEnabled(),
+        'infiniteScroll' => array(
+                'enabled' => $config->isInfiniteScrollEnabled(),
```

Comment:
I had translations nested here before but I moved them. You're right, I'll remove one level.